devlab/tests/generate_load.py

```diff
@@ -295,7 +295,7 @@ def update_filtering_file(self):
     def create_flavors(self):
         for flavor in self.config.flavors:
             fl = self.novaclient.flavors.create(**flavor)
-            if flavor.get('is_public') == False:
+            if flavor.get('is_public') is False:
```

Comment:
Maybe `if not flavor["is_public"]:`? Is it really possible for flavor not to have `is_public` field?